### PR TITLE
ovs_host_vlan: fix netperf issue and kill vm at the end of the test

### PR DESCRIPTION
**Pull request by PaulYuuu**
Fix netperf issue and kill vm at the end of the test.

ID: 1770060
Signed-off-by: Yihuang Yu <yihyu@redhat.com>

### CODE REVIEW

**Comment by PaulYuuu** [comment]
Test result:
 (1/1) Host_RHEL.m8.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.ppc64le.io-github-autotest-qemu.ovs_host_vlan: PASS (1169.52 s)

**Comment by PaulYuuu** [comment]
@pezhang @yanglei-rh Please help to review, thanks.

**Comment by yanglei-rh** [comment]
LGTM and Test pass
(1/1) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.x86_64.io-github-autotest-qemu.ovs_host_vlan.q35: PASS (1058.09 s)

**Comment by PaulYuuu** [comment]
@vivianQizhu Could you take a time to review and merge it? thanks.

File: qemu/tests/ovs_host_vlan.py [hunk 1]
```diff
@@ -306,5 +307,8 @@ def run(test, params, env):
                                                s_session=sessions[0],
                                                d_session=sessions[1],
                                                file_transfer_timeout=f_tmout)
+        for session in sessions:
+            if session:
+                session.close()
```
Comment (vivianQizhu) [inline qemu/tests/ovs_host_vlan.py]: What about move it to `finally:`, in case it fails some where?

Comment (PaulYuuu) [reply qemu/tests/ovs_host_vlan.py]: I am worried if case failed before creating sessions, move this to `finally` but in fact `sessions` have not yet been defined.

Comment (vivianQizhu) [reply qemu/tests/ovs_host_vlan.py]: What about move `sessions = []` before `try`

Comment (PaulYuuu) [reply qemu/tests/ovs_host_vlan.py]: Sounds good to me

**Comment by vivianQizhu** [comment]
Ack.

**Pull request merged**